apps/content/api/public/recitation_track_list.py

```
@@ -34,7 +34,7 @@ class RecitationSurahTrackOut(Schema):
     ayahs_timings: list[RecitationAyahTimingOut]
 
 
-@router.get("recitations/{asset_id}/", response=list[RecitationSurahTrackOut])
+@router.get("recitation-tracks/{asset_id}/", response=list[RecitationSurahTrackOut])
 @paginate
 def list_recitation_tracks(request: Request, asset_id: int):
```

Ruff (0.14.13)

Unused function argument: `request`

(ARG001)